internal/cmd/ske/kubeconfig/login/login.go

```diff
@@ -0,0 +1,240 @@
+package login
+
+import (
+	"context"
+	"crypto/sha256"
+	"crypto/x509"
+	"encoding/json"
+	"encoding/pem"
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/cache"
+	"k8s.io/client-go/rest"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/ske/client"
+
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-sdk-go/services/ske"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/scheme"
+	clientauthenticationv1 "k8s.io/client-go/pkg/apis/clientauthentication/v1"
+	"k8s.io/client-go/tools/auth/exec"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+const (
+	expirationSeconds     = 30 * 60          // 30 min
+	refreshBeforeDuration = 15 * time.Minute // 15 min
+)
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "login",
```

Comment:
```suggestion
		Use:   "login",
		Hidden: true,
```

Since this command is only to be used inside the "special" kubeconfig, I suggest that we hide it from the CLI documentation.

We are planning to introduce CLI plugins in the near future, when we have that in place we could move this command there, it would be more appropriate

Reply:
As discussed: we won't hide this command to allow easier discovery of the login feature.